code-challenges.test.js

```diff
@@ -64,4 +137,11 @@ const accountTransactions2 = [250, -89, 100, -96]
 const accountTransactions3 = []
 // Expected output: []
 
-// b) Create the function that makes the test pass.
+// b) Create the function that makes the test pass
+
+const addAccounts = (array) => {
+  return array.map(accountTransactions1[0+1+2+3])
+  return array.map(accountTransactions2[0+1+2+3])
+  return array.map(accountTransactions3[0+1+2+3])
+
+}
```

Comment:
So yeah if you check out the [syllabus section on map](https://github.com/learn-academy-2022-golf/Syllabus/blob/main/javascript/higher-order-functions.md#map) you'll find that it requires usage of an anonymous function as an argument, called a callback function. This function has it's own parameters and logic that need to be passed in to allow for a return per item of the array.